index.md

```diff
@@ -1,57 +1,198 @@
 ---
 _appTitle:
 _title: MonoGame
+_overrideDocFx: true
 ---
-<div id="big-image-header" />
-<div id="big-image-slogan">
-	<h1>One framework for creating<br/>powerful cross-platform games</h1>
-</div>
-<br/>
-<div id="big-image-quick-start">
-	<a href="/articles/index.html">Get started</a>
-	<a id="fork-button" href="https://github.com/mono/MonoGame">Fork us</a>
-</div>
-<br/>
-<div id="big-image-title">
-	<a id="built-with-monogame" href=""><img id="big-image-logo" src=""/></a><br/>
-	<a id="built-with-monogame" href="https://twitter.com/search?q=BuiltWithMonoGame">#BuiltWithMonoGame</a>
-</div>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-
-<section class="features-showcase">
-<div class="container-xxl content-center">
-<section>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Managed Code</h4>
-	<p>By leveraging C# and other .NET languages on Microsoft and Mono platforms you can write modern, fast, and reliable game code.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Cross-Platform</h4>
-	<p>MonoGame currently supports Windows, MacOS, Linux, iOS/iPadOS, Android, PlayStation 4, PlayStation 5, Xbox One, and Nintendo Switch.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Open-Source</h4>
-	<p>All the code is available to you ensuring you'll have the ability to make changes when you need to or even port to whole new platforms.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Community</h4>
-	<p>With 1000s of shipped games we have built up a vibrant community of developers which use MonoGame for both fun and profit.</p>
-</div>
+<section class="hero" >
+	<div id="hero-background-image" />
+	<div class="hero-content">

```

Comment:
So the darkening of the game hero image works well in dark mode:

![image](https://github.com/MonoGame/monogame.github.io/assets/103522/987f1261-9b51-497f-aee4-8491dbafaff4)

In light mode it lightens the image:

![image](https://github.com/MonoGame/monogame.github.io/assets/103522/084a9800-d5a0-423d-ad17-c4c8ef875725)

It seems to me like it should always darken even in the light mode for this particular case.

Reply:
This was actually supposed to be disabled and overlooked on my part. I initially added the opacity because the hero text was hard to read on some images.  I eventually switched it to use a text shadow to pop it out more, so the opacity can be removed.  

One thing I did notice however when removing the opacity, in relation to the other comment about the hash tag, it becomes even harder to read even though I've added the text shadow to it and bold styling.

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/314442b6-318f-42a6-8686-8821974f8b1b)


I'll add it here, but going to change that text to white so there are no issues with it blending with the background of any images now or added in the future to the heading section.